src/app/shared/components/timer/timer.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnDestroy } from '@angular/core';
+import { interval, Subscription } from 'rxjs';
+
+const rxTimer = interval(10);
+
+@Component({
+  selector: 'app-timer',
+  templateUrl: './timer.component.html',
+  styleUrls: ['./timer.component.scss'],
+})
+export class TimerComponent implements OnDestroy {
+  time = '0';
+  clock?: Subscription | null;
```

Comment:
stop$: Subject<void> = new Subject();
  clock$ = interval(10).pipe(takeUntil(this.stop$));